### PR TITLE
Nbala finance: dex fees and volume

### PR DESCRIPTION
**Pull request by yellowBirdy**

### CODE REVIEW

**Comment by llamatester** [comment]
The nabla adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs nabla
    
    🦙 Running NABLA adapter 🦙
    ---------------------------------------------------
    Start Date:	Sat, 02 Nov 2024 22:42:07 GMT
    End Date:	Sun, 03 Nov 2024 22:42:07 GMT
    ---------------------------------------------------
    
    ARBITRUM 👇
    Backfill start time: 15/8/2024
    Daily fees: 101
    └─ Methodology: All fees comes from the user.
    Daily protocol revenue: 0
    └─ Methodology: Currently no fees are taken by the protocl.
    Daily user fees: 101
    └─ Methodology: User pays between 0.01% and 0.1% fees on each swap.
    Daily volume: 202.80 k
    End timestamp: 1730673726 (2024-11-03T22:42:06.000Z)
    
    
    BASE 👇
    Backfill start time: 12/9/2024
    Daily fees: 0
    └─ Methodology: All fees comes from the user.
    Daily revenue: 0
    Daily user fees: 0
    └─ Methodology: User pays between 0.01% and 0.1% fees on each swap.
    Daily volume: 0
    End timestamp: 1730673726 (2024-11-03T22:42:06.000Z)

File: dexs/nabla/index.ts [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        const volumeLogs = await Promise.all(routers.map(router => getLogs({
+                target: router,
+                eventAbi: abis.router.swapEvent
+            })
+        ));
+
+        volumeLogs.forEach((logs, i) => {
+            logs.forEach((e: any) => {
+                dailyVolume.add(e.tokenIn, e.amountIn)
+                dailyVolume.add(e.tokenOut, e.amountOut)
```
Comment (dtmkeng) [inline dexs/nabla/index.ts:81]: volume should count one side, i think should count only tokenOut

File: dexs/nabla/index.ts [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            
+            const dailyVolume = createBalances()
+            const volumeLogs = await getLogs({
+                target: router,
+                eventAbi: abis.router.swapEvent
+            });
+    
+            volumeLogs.forEach((e, i) => {
+                dailyVolume.add(e.tokenIn, e.amountIn)
+                dailyVolume.add(e.tokenOut, e.amountOut)
```
Comment (dtmkeng) [inline dexs/nabla/index.ts]: volume should count one side, i think should count only tokenOut

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The nabla adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs nabla
    
    🦙 Running NABLA adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 05 Nov 2024 11:19:51 GMT
    End Date:	Wed, 06 Nov 2024 11:19:51 GMT
    ---------------------------------------------------
    
    ARBITRUM 👇
    Backfill start time: 15/8/2024
    Daily fees: 276
    └─ Methodology: All fees comes from the user.
    Daily protocol revenue: 0
    └─ Methodology: Currently no fees are taken by the protocl.
    Daily user fees: 276
    └─ Methodology: User pays between 0.01% and 0.1% fees on each swap.
    Daily volume: 551.35 k
    End timestamp: 1730891990 (2024-11-06T11:19:50.000Z)
    
    
    BASE 👇
    Backfill start time: 12/9/2024
    Daily fees: 18
    └─ Methodology: All fees comes from the user.
    Daily revenue: 0
    Daily user fees: 18
    └─ Methodology: User pays between 0.01% and 0.1% fees on each swap.
    Daily volume: 0
    End timestamp: 1730891990 (2024-11-06T11:19:50.000Z)

**Pull request merged**